package.json

```diff
@@ -7,7 +7,7 @@
     "@emotion/react": "^11.10.4",
     "@emotion/styled": "^11.10.4",
     "@fontsource/rubik": "^4.5.11",
-    "@liftedinit/many-js": "github:liftedinit/many-js",
+    "@liftedinit/many-js": "file:../many-js",
```

Comment:
I'll merge https://github.com/liftedinit/many-js/pull/106 and then change this back.